### ISSUE TITLE
Fresh tubesync Instance stopped working again

### ISSUE THREAD

**Issue by neuleo** [meeb/tubesync#483]
Hey, I recently opened an Issue, because yt-dlp returned Errors when indexing and afterward completely stopped downloading even when I press index. You answered and said this will be fixed with the new release. I updated my Docker Container and tested it. Unfortunate, it still wouldn't work. So I decided to set up a new, fresh tube sync Instance. It worked for a few Days, but since the 4th it stopped indexing and downloading again. When I press index, the HTTP request shows, that I pressed the Button, but nothing happens afterward.

**Comment by meeb** [comment]
What button specifically? What errors, if any, are in your container logs? Any errors on your "tasks" page?

**Comment by neuleo** [comment]
What button specifically:
![image](https://github.com/meeb/tubesync/assets/99101285/c393714c-071b-4983-b207-cda9505cff96)

What errors, if any, are in your container logs:
There are no Errors in my Logs

Any errors on your "tasks" page:
Yes, 
![image](https://github.com/meeb/tubesync/assets/99101285/5cd28446-8f46-405e-b9a1-6a1b87d75d00)

**Comment by meeb** [comment]
That playlist URL works fine for me in testing. The error you're getting basically means "unable to get info on the URL" which means either `yt-dlp` can't connect to YouTube, YouTube has blocked or throttled your IP, your playlist is invalid or requires authentication, etc. It's a generic catch-all error.

Make sure your workers are set to 1 (the default), anything other than 1 if you've changed this can cause you to get throttled. You can also debug this by invoking `yt-dlp` directly with:

```bash
$ docker exec -ti tubesync python3 /app/manage.py youtube-dl-info "https://www.youtube.com/playlist?list=PLhaT7JExJqAiX4Pp5tL1LgnM9SnpKYm5A"
```

If that works properly it'll return a big blob of JSON to the terminal.

Basically, pretty sure this isn't anything to do with tubesync.

**Comment by neuleo** [comment]
Just Using the URL works for me as well. But the sync in tube sync not. The Playlist is "not listed", but public. I haven't changed much in my docker compose file, and just checked. I haven't defined the Worker via the environment variables, so it should use the default 1 Worker.

I ran the python command and the result was the big blob of JSON with the Infos about the Videos in the Playlist. At the end it said "done". So yt-dlp can access the Playlist.

**Comment by neuleo** [comment]
The Container Log doesn't show any Error.
![image](https://github.com/meeb/tubesync/assets/99101285/572ca607-a482-483c-915a-985e2a5d50d1)

These are the Logs, when I force a Sync with the Playlist:
192.168.178.128 - - [11/Mar/2024:11:05:46 +0100] "GET / HTTP/1.1" 200 3727 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:46 +0100] "GET /static/styles/tubesync.css HTTP/1.1" 200 34554 "http://192.168.178.11:809/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:46 +0100] "GET /static/fonts/roboto/roboto-regular.woff HTTP/1.1" 304 0 "http://192.168.178.11:809/static/styles/tubesync.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:46 +0100] "GET /static/fonts/fontawesome/fa-solid-900.woff2 HTTP/1.1" 304 0 "http://192.168.178.11:809/static/styles/tubesync.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:46 +0100] "GET /static/fonts/fontawesome/fa-regular-400.woff2 HTTP/1.1" 304 0 "http://192.168.178.11:809/static/styles/tubesync.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:46 +0100] "GET /static/fonts/roboto/roboto-bold.woff HTTP/1.1" 304 0 "http://192.168.178.11:809/static/styles/tubesync.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:46 +0100] "GET /static/fonts/fontawesome/fa-brands-400.woff2 HTTP/1.1" 304 0 "http://192.168.178.11:809/static/styles/tubesync.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:47 +0100] "GET /static/fonts/roboto/roboto-light.woff HTTP/1.1" 304 0 "http://192.168.178.11:809/static/styles/tubesync.css" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:05:49 +0100] "GET /sources HTTP/1.1" 200 2665 "http://192.168.178.11:809/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:06:04 +0100] "GET /source-sync-now/1824d4a3-5812-45ff-9d33-3129d14229cb HTTP/1.1" 302 0 "http://192.168.178.11:809/sources" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
192.168.178.128 - - [11/Mar/2024:11:06:04 +0100] "GET /sources?message=source-refreshed HTTP/1.1" 200 2733 "http://192.168.178.11:809/sources" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36"
100.125.30.134 - - [11/Mar/2024:11:06:18 +0100] "GET / HTTP/1.1" 200 15556 "-" "Uptime-Kuma/1.23.11"


Also, the Playlist is not the only Source that is not Syncing. I have 7 Sources. 6 Channels and 1 Playlist and no channel or Playlist is syncing

**Comment by meeb** [comment]
The container logs will have errors in when the tasks that's failing runs. Nothing in your log extract shows any tasks running, that's just GET requests the UI. I would assume the error would appear after that log snippet.

**Comment by neuleo** [comment]
Yes, but the Problem is I can't get it to run the Tasks. Even if I press run in the WebUI, only GET requests appear. The tasks for yt-dlp don't start.

**Comment by brpaz** [comment]
I have the same problem. Download works manually on youtube-dlp.

And the jobs are stuck. It´s not downloading anything for a few hours already and have more than 200 jobs pending.

![tubesync_errors](https://github.com/meeb/tubesync/assets/184563/e7704653-5378-48c9-91bd-74cc2e12c2a1)


The only logs I have are access logs from the healthcheck:

````
10.0.0.222 - - [24/Mar/2024:22:19:51 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"                               │
│ 10.0.0.222 - - [24/Mar/2024:22:20:01 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"                               │
│ 10.0.0.222 - - [24/Mar/2024:22:20:01 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"                               │
│ 10.0.0.222 - - [24/Mar/2024:22:20:11 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"                               │
│ 10.0.0.222 - - [24/Mar/2024:22:20:21 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"                               │
│ 10.0.0.222 - - [24/Mar/2024:22:20:31 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"                               │
│ 10.0.0.222 - - [24/Mar/2024:22:20:31 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"                               │
│ 10.0.0.222 - - [24/Mar/2024:22:20:41 +0000] "GET / HTTP/1.1" 200 15798 "-" "kube-probe/1.28"   
```

**Comment by meeb** [comment]
@brpaz are your playlists public and accessible? That error just means that yt-dlp didn't get any data for the request to index the playlist.

**Comment by meeb** [comment]
I'll close this for now as it lacks enough information to assist further. Feel free to comment back and I'll re-open it if you are still experiencing this issue.
